doc/changes/latest.rst

```diff
@@ -13,126 +15,152 @@ HIGHLIGHTS
  | **Support for Python 3.8 has been dropped.**
  | **We recommend upgrading to Python 3.11 or above.**
  |
+ | **This version is the first one to support Python 3.13.**
+ |
  | **Minimum supported versions of the following packages have been bumped up:**
  | - numpy -- 1.22.4
  | - nibabel -- 5.2.0
  | - scikit-learn -- 1.4.0
  | - joblib -- 1.2.0
  | - pandas -- 2.2.0
 
-NEW
----
+-   This version includes a new "surface" API that allow to represent surface meshes and data.
+
+-   Single hemisphere meshes can be instantiated with
+    :class:`~nilearn.surface.SurfaceMesh`, :class:`~nilearn.surface.InMemoryMesh` or :class:`~nilearn.surface.FileMesh`.
+    :class:`~nilearn.surface.SurfaceMesh`
+
+-   :class:`~nilearn.surface.PolyMesh` (and :class:`~nilearn.surface.PolyData`)
+    can be used to instantiate mesh (and respectively data) objects with both hemisphere.
+
+-   :class:`~nilearn.surface.SurfaceImage` are objects that hold meshes and data for both hemispheres.
+
+-   :class:`~nilearn.surface.SurfaceImage`, :class:`~nilearn.surface.PolyMesh`, :class:`~nilearn.surface.PolyData`
```

Comment:
If we provide dicts with "left" and "right" keys with paths to the gift files,  and then pass them into mesh and data variables of SurfaceImage, that would work too.

Reply:
I'd say this should be the most recommended way to load surface data

Reply:
As shown here: https://nilearn.github.io/dev/auto_examples/00_tutorials/plot_surface_101.html#load-the-surface-image

Reply:
actually I meant to point to the example you created so users can directly see it in action